documentation/assemblies/upgrading/assembly-upgrade-kafka.adoc

```diff
@@ -33,34 +48,14 @@ You can access the CRDs from {ReleaseDownload} or find them in the `crd` subdire
 . Upgrade all Kafka brokers and client applications to the latest supported Kafka version.
 +
 * xref:assembly-upgrading-kafka-versions-{context}[]
-* xref:con-strategies-for-upgrading-clients-{context}[] 
+* xref:con-strategies-for-upgrading-clients-{context}[]
 
-. If applicable, perform the following tasks:
-+
-.. Update existing custom resources to handle deprecated custom resource properties
-+
-* xref:assembly-upgrade-resources-{context}[]
-+
-NOTE: Custom resources can also be updated _before_ the Kafka upgrade.
-+
-.. Update listeners to use the `GenericKafkaListener` schema
+. If applicable, update listeners to use the `GenericKafkaListener` schema
```

Comment:
I wonder if this is some left over ... the listener upgrade is now part of the CRD upgrades. So it does not seem to be necessary to mention it here like this on its own.

Reply:
I've removed this step and moved the _Updating listeners to the generic listener configuration_ section under _Strimzi custom resource upgrades_ after the related _Upgrading Kafka resources to support v1beta2_ section